scripts/asn_edit

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import argparse
+from ..associations import asn_edit
```

Comment:
scripts require absolute import paths, since they are run outside of the package context.